java/memory/src/main/java/io/netty/buffer/ArrowBuf.java

```diff
@@ -1165,6 +1165,21 @@ public ArrowBuf setZero(int index, int length) {
     return this;
   }
 
+  /**
+   * Sets all bits to one in the specified range.
+   * @param index index index (0 based relative to the portion of memory
+   *              this ArrowBuf has access to)
+   * @param length length of bytes to set.
+   * @return this ArrowBuf
+   */
+  public ArrowBuf setOne(int index, int length) {
+    if (length != 0) {
+      this.checkIndex(index, length);
```

Comment:
this is redundant?

Reply:
I am afraid not.
It needs to check the range [index, index + length) is within the boundary of the ArrowBuf.

It also mirrors the implementation of setZero above.